### PR TITLE
fix(StressChaos): containerNames problem

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

### What problem does this PR solve?

Unify the `containerNames` field on front-end and back-end.

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

* PR to update `chaos-mesh/website`/`chaos-mesh/website-zh`:
* Need to update Chaos Dashboard component, related issue:
* Need to cheery-pick to the release branch

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

File: pkg/apiserver/experiment/experiment.go [hunk 1]
```diff
@@ -373,16 +373,13 @@ func (s *Service) createStressChaos(exp *core.ExperimentInfo, kubeCli client.Cli
 					Mode:     v1alpha1.PodMode(exp.Scope.Mode),
 					Value:    exp.Scope.Value,
 				},
+				ContainerNames: exp.Target.StressChaos.ContainerNames,
```
Comment (cwen0) [inline pkg/apiserver/experiment/experiment.go:376]: Panic here if `exp.Target.StressChaos.CintainerName` is nil.

Comment (g1eny0ung) [reply pkg/apiserver/experiment/experiment.go:376]: Will be an empty array because of the `omitempty` option?

Comment (cwen0) [reply pkg/apiserver/experiment/experiment.go:376]: Got it

Comment (cwen0) [reply pkg/apiserver/experiment/experiment.go:376]: you are right

**Comment by Muscule** [comment]
When this fix will be available?

**Comment by cwen0** [comment]
> When this fix will be available?

v2.0.1 included this fix. You can upgrade to v2.0.1 https://github.com/chaos-mesh/chaos-mesh/releases/tag/v2.0.1

**Pull request merged**